chromium_src/chrome/browser/ui/views/download/download_ui_context_menu_view.h

```diff
@@ -6,6 +6,9 @@
 #ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_DOWNLOAD_DOWNLOAD_UI_CONTEXT_MENU_VIEW_H_
 #define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_DOWNLOAD_DOWNLOAD_UI_CONTEXT_MENU_VIEW_H_
 
+#include "chrome/browser/download/download_ui_model.h"
```

Comment:
this is creating unnecessary dependencies, you should use forward declarations when possible, but see https://github.com/brave/brave-core/pull/32211/files#r2508570682 because these are not needed at all in the header